src/js/workshop-comments.js

```diff
@@ -0,0 +1,132 @@
+/* global convertDate */
+
+const commentForm = document.getElementById("comment-form");
+
+commentForm.addEventListener("submit", (event) => {
+	event.preventDefault();
+
+	const nameDomElement = document.getElementById("name");
+	const name = nameDomElement.value.trim();
+	const commentDomElement = document.getElementById("comment");
+	const comment = commentDomElement.value.trim();
+
+	const submitFailureMessage = document.querySelector(".submit-failure-message");
+
+	// Hide required and comment submission message.
+	// Note: It's necessary to hide "required" and "succesful/failed" submitted comment message indicators programmatically even though they have been hidden initially by CSS
+	// in the case the same user submits more comments after submitting an initial comment.
+	document.querySelectorAll(".required, .submit-success-message, .submit-failure-message").forEach(element => {
```

Comment:
".required" selector can be removed as those DOM elements are also be identified as ".required-name" and ".required-comment".